docs/20-web/20-development/10-conventions.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Conventions'
+sidebar_position: 1
+id: conventions
+---
+
+
+
+This is a collection of tips and conventions to follow when working on the [OpenCloud Web frontend](https://github.com/opencloud-eu/web).
+Since it is a living document, please open a PR if you find something missing.
+
+## Contributing to OpenCloud Web
+
+Everyone is invited to contribute. Simply fork the [codebase](https://github.com/opencloud-eu/web/),
+check the [issues](https://github.com/opencloud-eu/web/issues?q=is%3Aopen+is%3Aissue+label%3ATopic%3Agood-first-issue)
+for a suitable one and open a pull request!
+
+### Linting and Tests
+
+To make sure your pull request can be efficiently reviewed and won't need a lot of changes down the road, please run the linter and
+the unit tests via `pnpm lint --fix` and `pnpm test:unit` locally. Our [CI](https://drone.opencloud.eu/opencloud/web) will run on
+pull requests and report back any problems after that. For a further introduction on how we handle testing, please head to
+the [testing docs](./../30-testing/intro.md).
+
```

Comment:
We should also add a sentence about formatting, since this is a new command that's also running in CI.